sdbuild/Makefile

```diff
@@ -52,6 +52,7 @@ $(BUILD_ROOT)/PYNQ: | $(BUILD_ROOT)
 
 $(PYNQ_UPDATE): | $(BUILD_ROOT)/PYNQ
 	cd $(BUILD_ROOT)/PYNQ && git fetch origin && git checkout $(PYNQ_MASTER_COMMIT)
+	cp -rf $(BOARDDIR)/* $(BUILD_ROOT)/PYNQ/boards/
```

Comment:
I'd prefer for this to be on a board-by-board basis - this has the potential to cause issue if BOARDDIR is the default and there are unchecked/untracked changes. I think packages/pynq/pre.sh might be a better place as we delete pynq_git in the image after install anyway.